lib/graph/graph.js

```diff
@@ -29,8 +29,8 @@ const log = getLogger("generateProjectGraph");
  * @param {string} [options.versionOverride] Framework version to use instead of the one defined in the root project
  * @param {string} [options.resolveFrameworkDependencies=true]
  * 		Whether framework dependencies should be added to the graph
- * @param {string} [options.workspaceName]
- * 		Name of the workspace configuration that should be used if any is provided
+ * @param {string} [options.workspaceName=default]
```

Comment:
We should also make clear that with `null` the usage of the default workspace can be skipped. The CLI currently already passes `null` if `--no-workspace` option is set.

```suggestion
 * @param {string|null} [options.workspaceName=default]
```